src/topology/metric_space/thickened_indicator.lean

```diff
@@ -251,3 +244,38 @@ begin
 end
 
 end thickened_indicator -- section
+
+section indicator
+
+variables {α : Type*} [pseudo_emetric_space α]
+
+lemma tendsto_indicator_thickening_indicator_closure (f : α → ℝ≥0∞) {δs : ℕ → ℝ}
+  (δs_pos : ∀ n, 0 < δs n) (δs_lim : tendsto δs at_top (𝓝 0)) (E : set α) :
+  tendsto (λ n, (metric.thickening (δs n) E).indicator f) at_top (𝓝 (indicator (closure E) f)) :=
```

Comment:
You can state
```lean
tendsto (λ δ, (metric.thickening δ E).indicator f) (𝓝[>] 0) (pure $ indicator (closure E) f)
```
and prove it using an order-dual version of `antitone.tendsto_indicator` and `map_coe_Ioi_at_bot`

Reply:
Also, I would move this lemma to a new PR.

Reply:
The generalization to the `nhds_within` filter seems like a good idea.

But `antitone.tendsto_indicator ` is for the `at_top` filter, so can't directly be applied if one generalizes as you suggested.

I will anyway move this to a different PR.